solcore/data_analysis_tools/ellipsometry_analysis.py

```diff
@@ -129,14 +130,14 @@ def plot(self, log=True):
 class EllipsometryFitter:
     """ Class that contains all the tools to fit the properties of a OptiStack model to ellipsometry data."""
 
-    def __init__(self, data, stack=None, vars=None, wavelength_range=None):
+    def __init__(self, data, stack=None, vars=None, wavelength_range=None) -> None:
```

Comment:
Considering what this constructor looks like, this class can probably be made a [`dataclass`](https://docs.python.org/3/library/dataclasses.html). But that's another story, nothing to change now.